next-app/src/routes/password/change/+page.ts

```diff
@@ -1 +0,0 @@
-export const prerender = true
```

Comment:
I'm curious why this got deleted 🙂

Reply:
constraint of utilizing the "action" paradigm, some early discussion here: [Prerendered page alongside non-GET request handler should be an error](https://github.com/sveltejs/kit/issues/4252#top).

doc reference: https://kit.svelte.dev/docs/page-options#prerender-when-not-to-prerender

> Pages with [actions](https://kit.svelte.dev/docs/form-actions) cannot be prerendered, because a server must be able to handle the action POST requests.